### PR TITLE
[docs] update docs to reflect updated code using 128MiB

### PR DESCRIPTION
**Pull request by halsaphi**
## Change Description

Update to docs - zero impact to code base. 

## Change Type
**Select ONE**
- [x ] Documentation

- [ ] Stability bug fix

- [ ] Other

- [ ] Other - special case

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [x ] Documentation Additions

### CODE REVIEW

**Review by iamveritas — APPROVED** [review]

File: docs/01_nodeos/02_usage/60_how-to-guides/10_how-to-configure-state-storage.md [hunk 1]
```diff
@@ -15,14 +15,14 @@ Config Options for eosio::chain_plugin:
                                         compaction. Defaults to the number of available cores. 
   --rocksdb-files arg (=-1)             Max number of rocksdb files to keep 
                                         open. -1 = unlimited.
-  --rocksdb-write-buffer-size-mb arg (=134217728)
-                                        Size of a single rocksdb memtable
+  --rocksdb-write-buffer-size-mb arg (=128MB)
```
Comment (heifner) [inline docs/01_nodeos/02_usage/60_how-to-guides/10_how-to-configure-state-storage.md:18]:
```
  --rocksdb-write-buffer-size-mb arg (=128)
                                        Size of a single rocksdb memtable (in MiB)
```

File: docs/01_nodeos/03_plugins/chain_plugin/index.md [hunk 2]
```diff
@@ -120,8 +120,8 @@ Config Options for eosio::chain_plugin:
 										available cores.  
   --rocksdb-files arg (=-1)             Max number of rocksdb files to keep 
                                         open. -1 = unlimited.
-  --rocksdb-write-buffer-size-mb arg (=134217728)
-                                        Size of a single rocksdb memtable
+  --rocksdb-write-buffer-size-mb arg (=128MB)
+                                        Size of a single rocksdb memtable (in MiB)
```
Comment (heifner) [inline docs/01_nodeos/03_plugins/chain_plugin/index.md:124]:
```
  --rocksdb-write-buffer-size-mb arg (=128)
                                        Size of a single rocksdb memtable (in MiB)
```
If you specify `128MB` on the command line that will not work.

**Review by heifner — CHANGES_REQUESTED** [review]

**Comment by allenhan2** [comment]
"Philip Halsall" doesn't have permission to trigger builds in "EOSIO Sync from Genesis",   
could any sponsor of granting the permission to fire a service ticket? thanks,

File: docs/01_nodeos/03_plugins/chain_plugin/index.md (hunk 2 above)
Comment (halsaphi) [reply docs/01_nodeos/03_plugins/chain_plugin/index.md:124]: Removed MB from the doc, now only in description.

**Review by lparisc — CHANGES_REQUESTED** [review]
Few changes:
- Wrap the config options in codeblocks \`\`\` ... \`\`\` to render properly. It currently renders like this:
"--backing-store arg (=chainbase) The storage for state, chainbase or rocksdb --rocksdb-threads arg Number of rocksdb threads for flush and compaction. Defaults to the number of available cores. --rocksdb-files arg (=-1) Max number of rocksdb files to keep open. -1 = unlimited. --rocksdb-write-buffer-size-mb arg (=128) Size of a single rocksdb memtable (in MiB)"
- Complete "Next Steps" section or remove title for now.
- `docs/01_nodeos/02_usage/index.md` must be updated to include the new `60_how-to-guides` section

File: docs/01_nodeos/03_plugins/chain_plugin/index.md (hunk 2 above)
Comment (gssagoo) [reply docs/01_nodeos/03_plugins/chain_plugin/index.md:124]: Looks good passing just the 128 without the MB.

**Review by gssagoo — APPROVED** [review]
Looks good.

**Review by heifner — APPROVED** [review]

File: docs/01_nodeos/02_usage/60_how-to-guides/10_how-to-configure-state-storage.md [hunk 3]
```diff
@@ -7,22 +7,29 @@ Version 2.1 or above of the EOSIO development environment.
...
 Config Options for eosio::chain_plugin:
 
   --backing-store arg (=chainbase)      The storage for state, chainbase or 
                                         rocksdb
-  --rocksdb-threads arg 	            Number of rocksdb threads for flush and
-                                        compaction. Defaults to the number of available cores. 
-  --rocksdb-files arg (=-1)             Max number of rocksdb files to keep 
+  --persistent-storage-num-threads arg 	Number of rocksdb threads for flush and
+                                        compaction.  Defaults to the number of 
+										available cores.
```
Comment (heifner) [inline docs/01_nodeos/02_usage/60_how-to-guides/10_how-to-configure-state-storage.md:17]:
```  
--persistent-storage-num-threads arg (=1)
                                        Number of rocksdb threads for flush and
                                        compaction
```

File: docs/01_nodeos/03_plugins/chain_plugin/index.md [hunk 4]
```diff
@@ -115,13 +115,17 @@ Config Options for eosio::chain_plugin:
                                         drops below this size (in MiB).
   --backing-store arg (=chainbase)      The storage for state, chainbase or 
                                         rocksdb
-  --rocksdb-threads arg 	            Number of rocksdb threads for flush and
+  --persistent-storage-num-threads arg 	Number of rocksdb threads for flush and
                                         compaction.  Defaults to the number of 
 										available cores.
```
Comment (heifner) [inline docs/01_nodeos/03_plugins/chain_plugin/index.md:120]: No longer defaults to number of cores.

**Review by heifner — CHANGES_REQUESTED** [review]

**Review by gssagoo — APPROVED** [review]
The changes to removing MiB from the --persistent-storage-write-buffer-size-mb parameter looks good.  Also the comment change reflecting that we no longer default to the number of cores for --persistent-storage-num-threads looks good.  thanks

File: docs/01_nodeos/02_usage/60_how-to-guides/10_how-to-configure-state-storage.md (hunk 1 above)
Comment (halsaphi) [reply docs/01_nodeos/02_usage/60_how-to-guides/10_how-to-configure-state-storage.md]: updated

File: docs/01_nodeos/03_plugins/chain_plugin/index.md (hunk 2 above)
Comment (halsaphi) [reply docs/01_nodeos/03_plugins/chain_plugin/index.md]: updated

**Comment by halsaphi** [comment]
re "docs/01_nodeos/02_usage/index.md must be updated to include the new 60_how-to-guides section" this doesn't match cleos and keod sections though ... should this be discussed?

**Review by iamveritas — APPROVED** [review]

**Review by lparisc — APPROVED** [review]

**Review by heifner — APPROVED** [review]

**Pull request merged**